Dockerfile

```diff
@@ -1,27 +1,20 @@
-FROM ubuntu:22.04 as builder
+FROM golang:1.19-bullseye as builder
 
 RUN apt-get update && \
-    apt-get install -y --no-install-recommends curl git ca-certificates build-essential libelf-dev gnupg2
+    apt-get install -y --no-install-recommends build-essential libelf-dev gnupg2
```

Comment:
With `golang:1.19-bullseye` you don't need:

* `build-essential` as we have all we need in the base image
* `gnupg2` as we don't add any new repos with their own keys

It works with just `libelf-dev`, so let's only keep that.

Reply:
Done.